### DISCUSSION TITLE
Trovis 5573

### DISCUSSION THREAD

**Discussion by theMPstig** [Tom-Bom-badil/samson_trovis_557x#61] [General (de/en)]
Hallo Tom,
ich habe eine 5573 und lese dein Projekt mit großem Interesse - bekomme das Löten jedoch nicht auf die Reihe. Besteht da eine Möglichkeit, dass du mir so etwas gegen eine entsprechenden Unkostenpauschale umsetzt?
Grüße

**Comment by Tom-Bom-badil** [comment]
Solved in shNG Forum - issue closed.
